packages/myst-to-jats/src/transforms/definitions.ts

```diff
@@ -0,0 +1,35 @@
+import type { Plugin } from 'unified';
+import type { Root } from 'mdast';
+import type { Parent } from 'myst-spec';
+import { selectAll } from 'unist-util-select';
+
+export type DefinitionList = Parent & { type: 'definitionList' };
+export type DefinitionItem = Parent & { type: 'definitionItem' };
+export type DefinitionTerm = Parent & { type: 'definitionTerm' };
+export type DefinitionDescription = Parent & { type: 'definitionDescription' };
```

Comment:
I think we need a staging ground for mystjs-used types that are not in spec yet. These are copied from a few other places, and it would be nice to be consistent. @fwkoch thoughts on a library for that?

Reply:
Yeah, that would be good.

My first thought is to just chuck it into a folder in `myst-common`... but having a separate package is probably nicer so we can reference it independently (i.e. in `myst-spec` readme, MEPs, etc). `myst-spec-incubator` `myst-spec-proposals` `myst-spec-alpha`...